src/Header/ContentDisposition.php

```diff
@@ -137,26 +137,34 @@ public function getFieldValue($format = HeaderInterface::FORMAT_RAW)
                 }
             } else {
                 // Use 'continuation' per RFC 2231
-                $maxValueLength = strlen($value);
-                do {
-                    $maxValueLength = ceil(0.6 * $maxValueLength);
-                } while ($maxValueLength > self::MAX_PARAMETER_LENGTH);
-
                 if ($valueIsEncoded) {
-                    $encodedLength = strlen($value);
                     $value = HeaderWrap::mimeDecodeValue($value);
-                    $decodedLength = strlen($value);
-                    $maxValueLength -= ($encodedLength - $decodedLength);
                 }
 
-                $valueParts = str_split($value, $maxValueLength);
                 $i = 0;
-                foreach ($valueParts as $valuePart) {
-                    $attributePart = $attribute . '*' . $i++;
-                    if ($valueIsEncoded) {
-                        $valuePart = $this->getEncodedValue($valuePart);
+                $fullLenght = mb_strlen($value, 'UTF-8');
```

Comment:
```suggestion
                $fullLength = mb_strlen($value, 'UTF-8');
```

use correct spelling